ibm_watson/language_translator_v3.py

```diff
@@ -399,17 +464,118 @@ def list_models(self,
         if 'headers' in kwargs:
             headers.update(kwargs.get('headers'))
         sdk_headers = get_sdk_headers('language_translator', 'V3',
-                                      'list_models')
+                                      'list_documents')
         headers.update(sdk_headers)
 
-        params = {
-            'version': self.version,
-            'source': source,
-            'target': target,
-            'default': default_models
-        }
+        params = {'version': self.version}
 
-        url = '/v3/models'
+        url = '/v3/documents'
+        response = self.request(
+            method='GET',
+            url=url,
+            headers=headers,
+            params=params,
+            accept_json=True)
+        return response
+
+    def translate_document(self,
+                           file,
+                           filename=None,
```

Comment:
Filename should be required here since it will be generated as required. Hopefully we will update the generator to support making filename optional but today it has the same required as the file.

Reply:
Its already updated, this is how it gets generated in python
```       
 if not filename and hasattr(file, 'name'):
            filename = basename(file.name)
        if not filename:
            raise ValueError('filename must be provided')
```